news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+
+- [Nekrogoblikon - Mold](https://osu.ppy.sh/beatmapsets/1061153#osu/2221627) hosted by [Icekalt](https://osu.ppy.sh/users/5410645)
+- [Nekrogoblikon - The Skin Thief](https://osu.ppy.sh/beatmapsets/1061535) hosted by [LMT](https://osu.ppy.sh/users/7262798)
+- [Nekrogoblikon - Dragons](https://osu.ppy.sh/beatmapsets/1083931#osu/2266883) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+![Panda Eyes header](https://assets.ppy.sh/artists/55/header.jpg)
+
+For the **On My Own (osu!taiko)** quest, the mappers had to create and rank beatmap for "On My Own" by Panda Eyes
+
+This quest was completed by **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[Faputa](https://osu.ppy.sh/users/845733)**
+
+- [Panda Eyes - On My Own](https://osu.ppy.sh/beatmapsets/1065837) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
+
+For the **Panda Eyes solo** quests, each mapper had to create and rank 1 mapset in their mode of any song by Panda Eyes
+
+The **osu!catch** version of this quest was completed by **[Nelly](https://osu.ppy.sh/users/4741164)**
+
+- [Panda Eyes & Teminite - Immortal Flame (feat. Anna Yvette)](https://osu.ppy.sh/beatmapsets/1069737#fruits/2239249) by [Nelly](https://osu.ppy.sh/users/4741164)
+
+The **osu!mania** version of this quest was completed by **[Leniane](https://osu.ppy.sh/users/7138602)**
+
+- [Panda Eyes & Teminite - Highscore](https://osu.ppy.sh/beatmapsets/1079712#mania/2258948) by [Leniane](https://osu.ppy.sh/users/7138602)
+
+The **osu!** version of this quest was completed by **[Hazu-](https://osu.ppy.sh/users/4668230)**
+
+- [Panda Eyes - Pump It Up](https://osu.ppy.sh/beatmapsets/1090677#osu/2280138) by [Hazu-](https://osu.ppy.sh/users/4668230)
+
+![Ricky Montgomery header](https://assets.ppy.sh/artists/54/header.jpg)
+
+For the **Ricky Montgomery mini-pack (osu!)** quest, the mappers had to create and rank at least 3 mapsets of songs by Ricky Montgomery
+
+This quest was completed by **[Trynna](https://osu.ppy.sh/users/2652951)** and **[eiri-](https://osu.ppy.sh/users/3388410)**
+
+- [Ricky Montgomery - Cabo](https://osu.ppy.sh/beatmapsets/1081806) hosted by [Trynna](https://osu.ppy.sh/users/2652951)
+- [Ricky Montgomery - California](https://osu.ppy.sh/beatmapsets/1085943#osu/2270844) hosted by [eiri-](https://osu.ppy.sh/users/3388410)
+- [Ricky Montgomery - Snow](https://osu.ppy.sh/beatmapsets/1086534#osu/2272128) hosted by [eiri-](https://osu.ppy.sh/users/3388410)
+
+For the **Ricky Montgomery solo** quests, each mapper had to create and rank 1 mapset in their mode of any song by Ricky Montgomery
+
+The **osu!taiko** version of this quest was completed by **[Faputa](https://osu.ppy.sh/users/845733)**
```

Comment:
when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)